src/mistral_common/tokens/tokenizers/audio.py

```diff
@@ -178,16 +190,51 @@ def _encode_audio_chunk(self, content: AudioChunk) -> AudioEncoding:
             audio=audio,
         )
 
-    def __call__(self, content: AudioChunk) -> AudioEncoding:
-        r"""Call the encoder on an audio chunk.
+    def _encode_audio_chunk(self, content: AudioChunk) -> AudioEncoding:
+        audio = Audio.from_raw_audio(content.input_audio)
+        return self._encode_audio(audio)
+
+    def _encode_audio_url_chunk(self, content: AudioURLChunk) -> AudioEncoding:
+        url = content.audio_url if isinstance(content.audio_url, str) else content.audio_url.url
```

Comment:
why do we need to allow both cases (`url` and `AudioURL` object) if we always only need the url as a string?